entrypoint.sh

```diff
@@ -3,10 +3,12 @@
 
 # Wait for the DB to start accepting connections.
 for i in $(seq 5); do
+   sleep 10
    echo "Attempt $i"
-   node migrate.js && break || sleep 10
+   bin/migrate && break
 done
```

Comment:
I need to double check that the node:latest image has a MySQL client installed. I think I originally went with this because it didn't.